config/apps/day3.config.json

```diff
@@ -2,5 +2,8 @@
   "appType": "stochastic",
   "title": "Day 3 - Stochastic Model",
   "stochasticProp": "day3 stochastic value",
```

Comment:
Is there a ticket to remove these dummy props? I think all the apps have them in the config, and they exit in responseTypes, but don't get used anywhere...

Reply:
Will add one

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3759/Remove-dummy-props-from-app-state